### PR TITLE
v0.6.0

### PR DESCRIPTION
**Pull request by mobileoverlord**
Adds support for elixir 1.9

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1,2 @@
+elixir 1.9-otp-22
```
Comment (jjcarstens) [inline .tool-versions]: Did you mean to leave this `asdf` file in?

File: example/mix.exs [hunk 2]
```diff
@@ -19,10 +20,21 @@ defmodule Example.MixProject do
...
+      ]
+    ]
+  end
+
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:shoehorn, path: "../"}
+      {:shoehorn, path: "../"},
+      {:distillery, "~> 2.1", github: "mobileoverlord/distillery", branch: "release-mutable", override: true},
```
Comment (jjcarstens) [inline example/mix.exs]: Does the example app require using your fork and branch or can distillery be used directly like in the other mix files here?

Comment (mobileoverlord) [reply example/mix.exs]: ah, missed this one, thanks

File: example/mix.lock [hunk 3]
```diff
@@ -1,4 +1,4 @@
 %{
-  "artificery": {:hex, :artificery, "0.2.6", "f602909757263f7897130cbd006b0e40514a541b148d366ad65b89236b93497a", [:mix], [], "hexpm"},
-  "distillery": {:hex, :distillery, "2.0.2", "58e15f6708e355b610a48356ef170e63ec7753e35e215b107e1e87ed79c65170", [:mix], [{:artificery, "~> 0.2", [hex: :artificery, repo: "hexpm", optional: false]}], "hexpm"},
+  "artificery": {:hex, :artificery, "0.4.2", "3ded6e29e13113af52811c72f414d1e88f711410cac1b619ab3a2666bbd7efd4", [:mix], [], "hexpm"},
+  "distillery": {:git, "https://github.com/mobileoverlord/distillery.git", "7e73be84e0f7f8ac156c735bea6bc2f8f1d5636b", [branch: "release-mutable"]},
```
Comment (fhunleth) [inline example/mix.lock]: It looks like the example project needs a `mix deps.update distillery` since I assume that you don't want it using your fork?

**Pull request merged**